modules/rostests/apitests/ntdll/NtAllocateVirtualMemory.c

```diff
@@ -674,17 +724,20 @@ CheckSomeDefaultAddresses(VOID)
                                      &Size,
                                      MEM_RESERVE,
                                      PAGE_READWRITE);
+    // Result varies between Windows versions and sometimes between configurations/runs.
+    ok_ntstatus(Status, STATUS_SUCCESS);
     if (NT_SUCCESS(Status))
     {
-        trace("Below 0x00400000 is available, as on ReactOS and Windows S03\n");
+        trace("Below 0x00400000 is available, as on ReactOS and Windows XP/S03/8.1+_amd64. (Windows 8.1/10_x64 vary.)\n");
+        // ReactOS:
         // 0x003F0000, 64k: Free.
-        ok_ntstatus(Status, STATUS_SUCCESS);
         Status = NtFreeVirtualMemory(NtCurrentProcess(), &BaseAddress, &Size, MEM_RELEASE);
         ok_ntstatus(Status, STATUS_SUCCESS);
     }
     else
     {
-        trace("Below 0x00400000 is not available, as on Windows XP\n");
+        trace("Below 0x00400000 is not available, as on Windows Vista_amd64. (Windows Vista/S08_x64/7/S08_amd64/7_amd64 vary.)\n");
```

Comment:
S08_x64, S08_amd64? Those are the same
Also, "S08" sounds weird to me. "S2008" would make more sense, though it's still a bit odd.

Reply:
2k8 / 2k8 R2 would be best.